node/service/src/chain_spec.rs

```diff
@@ -1524,7 +1590,7 @@ pub fn rococo_testnet_genesis(
 fn polkadot_development_config_genesis(wasm_binary: &[u8]) -> polkadot::GenesisConfig {
 	polkadot_testnet_genesis(
 		wasm_binary,
-		vec![get_authority_keys_from_seed("Alice")],
+		vec![get_authority_keys_from_seed_no_beefy("Alice")],
```

Comment:
Because `with_beefy` is the new default, when we migrate `polkadot` and `kusama` to include BEEFY we are either stuck with `_with_beefy` methods or we need to rename later.